x-pack/plugins/cases/server/features.ts

```diff
@@ -38,6 +38,7 @@ export const getCasesKibanaFeature = (): KibanaFeatureConfig => {
     cases: [APP_ID],
     privileges: {
       all: {
+        api: ['casesSuggestAssignees'],
```

Comment:
nit: Maybe `suggestUserProfiles`?

Reply:
Good idea, how do you feel about keeping `cases` in there somewhere? I read in the `api` field comments that we need to avoid collisions with other plugins so I figured prefixing it with `cases` would help with that 🤷‍♂️

Reply:
oh yeah sorry, I forgot to add the `cases` prefix. Yes, keep it 🙂